ui/ducks/send/send.js

```diff
@@ -512,11 +513,12 @@ export const computeEstimatedGasLimit = createAsyncThunk(
     const isNonStandardEthChain = getIsNonStandardEthChain(state);
     const chainId = getCurrentChainId(state);
     const selectedAccount = getSelectedInternalAccountWithBalance(state);
+    const networkClientId = getSelectedNetworkClientId(state);
```

Comment:
As mentioned before, this argument isn't adding any multi-chain support yet since it just references the global network client ID.

So ideally it would be optional and only the `chainId` would be needed.

Reply:
Done.